docs/machine_learning/2_regression.md

```diff
@@ -0,0 +1,108 @@
+# Regression
+
+Regression, along with classification, are the most common machine learning problems.
```

Comment:
"the most common machine learning problems"

problems? Acho que queres algo como techniques ou algorithms.